gazebo_msgs/srv/SpawnModel.srv

```diff
@@ -1,6 +1,6 @@
 string model_name                 # name of the model to be spawn
 string model_xml                  # this should be an urdf or gazebo xml
-string model_namespace            # spawn model and all ROS interfaces under this namespace
+string robot_namespace            # spawn robot and all ROS interfaces under this namespace
```

Comment:
The message has been put back to its original status, please check the overall changes of this pull request